### DISCUSSION TITLE
Can we modify user data path?

### DISCUSSION THREAD

**Discussion by RazaGR** [coder/code-server#4242] [Q&A] [answered]
I have installed code-server using npm on alpine, I wanted to ask how could I  change these path? `~/.local/share/code-server`
`~/.config/code-server`
to my desired location? 
Thanks

**Comment by jsjoeio** [comment, marked as answer]
If you run `code-server --help`, you'll see the available options. One of those is `--config` which is "Path to yaml config file".

That should be what you're looking for!
